recipe/meta.yaml

```diff
@@ -253,6 +254,9 @@ outputs:
         - if [[ $($PYTHON -c 'import cv2; print(cv2.__version__)') != $PKG_VERSION ]]; then exit 1; fi  # [unix]
         - python -c "import cv2; assert 'Unknown' not in cv2.videoio_registry.getBackendName(cv2.CAP_V4L)"  # [linux]
         - python -c "import cv2, re; assert re.search('Lapack:\s+YES', cv2.getBuildInformation())"
+        - test $(pip list | grep opencv-python | tr -s " " | grep $PKG_VERSION | wc -l) -eq 1  # [unix]
```

Comment:
I personally think that the `tr` and the second `grep` commands are a little much.

Reply:
Cool, it was not working (I copied it from https://github.com/conda-forge/idyntree-feedstock/blob/master/recipe/meta.yaml#L70). Let's try to remove and check if it improves.

Reply:
i don't think it is the issue. I would add a `pip list` above just to see the output to help debug. It should help "visually test" on windows as well.

Reply:
I already did this in https://github.com/conda-forge/opencv-feedstock/pull/300/commits/cd3d201d06b23704dba7233835e82148ec2ad39a . : )